packages/utils/src/crypto.ts

```diff
@@ -71,3 +71,21 @@ export const networkNames: { [index: string]: string } = {
 export function isWellFormattedAddress(address: EthAddress): boolean {
   return /^(0x)?[0-9a-f]{40}$/i.test(address);
 }
+
+export async function isEthereumEnabled(): Promise<boolean> {
+  if ((window as any).ethereum) {
+    try {
+      return !!(await (window as any).ethereum.enable());
+    } catch (e) {
+      return false;
+    }
+  } else {
+    return true;
+  }
+}
+
+export async function enableEthereum(): Promise<any> {
```

Comment:
seems redundant with the other method above

Reply:
Yeah true. The only difference is that the above returns a boolean, the below returns a... well MM returns an array of one address. Will replace them both with a single `ethereumEnable` in my next PR.